### DISCUSSION TITLE
Hammerhead Matriarch and Tiger Thresher genes seem not to work on an uncounsious character

### DISCUSSION THREAD

**Discussion by Kokhirus** [FakeFishGames/Barotrauma#14063] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Tried to use Tiger Thresher + Hammerhead Matriarch genes as a husk substitude, but got the problem, that the character will suffocate even with 100% Thresher Genes.

### Reproduction steps

1. Give a character 100% Tiger Thresher gene
2. Knock him out
3. See low oxygen build up

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This is actually intentional: oxygenlow will always build up on unconscious characters until they die. Husks are a bit of a special case, they're the only ones that are able to spontaneously get up from being knocked out if you don't finish them off in time.

**Comment by Kokhirus** [comment]
> This is actually intentional: oxygenlow will always build up on unconscious characters until they die.

Oh, that's sad. I was really hoping there was a way that it is a bug, since a character with fully purified thresher gene doesn't require any oxygen and it would be cool to somehow prevent yourself from dying as if you are a husk, while being able to talk at the same time for an extra cost of having the 100% refined thresher gene. This would actually be pretty logical for 100% thresher gene, but this is not a bug discussion at this point. Thank you for your reply!
